demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -15,4 +15,6 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
         { type: "date", name: "availableSince", width: 140 },
         { type: "dateTime", name: "createdAt", width: 170 },
     ],
+    actionsWidth: 116,
+    actionsComponent: { name: "ProductsGridPreviewAction", import: "../../ProductsGridPreviewAction" },
```

Comment:
To me this sounds like I can replace the actions component altogether. Maybe add an `additional` prefix?